src/components/NavBar/styles.ts

```diff
@@ -54,7 +61,7 @@ export const NavBarContainer = styled.div<{ $topPosition?: number; $expandBreakp
     border-radius: 50% 50%;
     max-width: 3.785rem;
     height: auto;
-    box0px ${theme.grey70};
+    box-shadow: 0px 0px 4px 0px ${theme.grey30};
```

Comment:
I think this was a regression introduced in https://github.com/aserto-dev/aserto-react-components/pull/2